services/bettersocial/fcmToken/sendChatNotification.js

```diff
@@ -4,14 +4,25 @@ const {FcmToken, User} = require('../../../databases/models');
 const UsersFunction = require('../../../databases/functions/users');
 
 const sendChatNotification = async (userTargetId, payload) => {
-  const signedUser = await UsersFunction.findSignedUserId(User, userTargetId);
-  const userTargetToken = await FcmTokenFunction.findAllTokenByUserId(FcmToken, signedUser);
+  const signedUser = await UsersFunction.findSignedUserId(User, userTargetId, {
+    shouldReturnObject: true
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `findSignedUserId` function now accepts an options object with a `shouldReturnObject` property. This change could potentially introduce type inconsistency issues if not handled properly in the rest of the codebase. Make sure all calls to this function are updated accordingly.

```diff
- const signedUser = await UsersFunction.findSignedUserId(User, userTargetId);
+ const signedUser = await UsersFunction.findSignedUserId(User, userTargetId, {
+   shouldReturnObject: true
+ });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->